metrics/web/test/features/auth/domain/usercases/receive_current_user_updates_test.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:metrics/features/auth/domain/usecases/receive_current_user_updates.dart';
+import 'package:test/test.dart';
+
+import '../../../../test_utils/matcher_util.dart';
+
+void main() {
+  group("ReceiveCurrentUserUpdates", () {
```

Comment:
We should add tests for call method for all use cases - as currently it's not covered.